mongomock/store.py

```diff
@@ -104,3 +107,51 @@ def __len__(self):
     def documents(self):
         for doc in six.itervalues(self._documents):
             yield doc
+
+    def remove_expired_documents(self):
+        for index in six.itervalues(self.indexes):
+            if index.get('expireAfterSeconds') is not None:
```

Comment:
Nit - you could create a set of indexes that have an expireAfterSeconds field instead of checking them each time. I'm trying to optimize because this method is probably going to be called many many times.